README.md

```diff
@@ -12,117 +12,82 @@
+If you've already run partis to create this file, skip to `--run-linearham`; if not, you can have linearham run partis for you with `--run-partis`.
 
-The entire `linearham` pipeline can be executed using `scons`.
-
-To run `partis`:
+#### `--run-partis`
+This runs partis on an input sequence file:
 ```bash
-scons --run-partis --fasta-path=<string> [--all-clonal-seqs] --locus={igh|igk|igl} --parameter-dir=<string> --outdir=<string>
+scons --run-partis --fasta-path=<file> --locus={igh|igk|igl} --outdir=<dir> [--parameter-dir=<dir>]
 ```
-The `partis`-related command line arguments are described in the following table:
+Although `--fasta-path` can be any file type that partis `--infname` handles (see partis help).
+Partis requires a directory with fitted sample-specific parameters, which if not already present will be automatically inferred based on the sequences in the fasta file.
+Because linearham runs on only a single family at a time, but these parameters are more accurate if inferred on the entire repertoire of many clonal families, it is better if you use parameters cached in a previous partis run on the entire repertoire, and then pass them to linearham with `--parameter-dir`.
+However, if you don't, the automatically-inferred parameters will still work fine, they'll just be somewhat less accurate (since they'll only be based on the one family).
 
-| Command | Description |
+Other partis-related arguments:
+| Name | Description |
 | ---     | ---         |
-| `--fasta-path` | The repertoire FASTA file path. |
-| `--all-clonal-seqs` | Should the repertoire sequences be treated as clonal? |
-| `--locus` | Which immunoglobulin locus are we doing inference on (defaults to `igh`)? |
-| `--parameter-dir` | An optional directory of partis parameter files. |
+| `--all-clonal-seqs` | If set, attempts to force all sequences in the fasta file into the same clonal family; otherwise it runs partis partition to infer the clonal families |
+| `--locus` | Which immunoglobulin locus (defaults to `igh`)? |
 | `--outdir` | The output directory (defaults to `output`). |
 
-To run `linearham`:
+#### `--run-linearham`
+Once you have a partis output file, whether you made it separately or with linearham, you can run linearham itself:
+
 ```bash
-scons --run-linearham --partition-index=<int> --cluster-index=<int> --cluster-seed-unique-id=<str> --template-path=<string> --mcmc-iter=<int> --mcmc-thin=<int> --tune-iter=<int> --tune-thin=<int> --num-rates=<int> --burnin-frac=<double> --subsamp-frac=<double> --num-cores=<int> --rng-seed=<int> --lineage-unique-id=<string> --asr-pfilters=<double> --partis-yaml-file=<string> --parameter-dir=<string> --outdir=<string>
+scons --run-linearham --outdir=<dir> [--partis-yaml-file=<file>] [--parameter-dir=<dir>]
 ```
-The `linearham`-related command line arguments are described in the following table:
+If there is one cluster in the partis output file, linearham will run on that.
```

Comment:
I know you say 
> clonal families (clusters).

a few lines down, but I think we need to be more explicit about defining clusters as interchangeable with clonal families before we use the term cluster at all. This was probably true before you started editing but reading more carefully made me feel more strongly about this.